build.gradle

```diff
@@ -130,6 +130,8 @@ configurations.all {
         force "org.mockito:mockito-core:${versions.mockito}"
         force "org.yaml:snakeyaml:${versions.snakeyaml}"
         force "org.slf4j:slf4j-api:2.0.0"
+		force "ch.qos.logback:logback-classic:1.3.14"
+		force "ch.qos.logback:logback-core:1.3.14"
```

Comment:
indentation?

Reply:
Nice catch, fixed in [2d9efab](https://github.com/opensearch-project/observability/pull/1795/commits/2d9efabb939964ab538ac1d67b9a2c34d4285838). Thanks 🧑‍🚀